### PR TITLE
Capitalized "django" in docs

### PR DESCRIPTION
**Pull request by akki**

### CODE REVIEW

**Comment by timgraham** [comment]
We could remove "django" from docs/spelling_wordlist and surround things like "django-users" with backticks.

**Comment by akki** [comment]
There are many instances of "Django" in the documentation. as soon as I removed "django" from docs/spelling_wordlist all those instances threw an error when I run `make spelling`.
Am I missing something here?

**Comment by timgraham** [comment]
I'm not sure how things are working, but I didn't see errors besides instances of lower case "django". We can add the uppercase version of "Django" if needed.

File: docs/internals/contributing/committing-code.txt [hunk 1]
```diff
@@ -109,8 +109,8 @@ Committing guidelines
 In addition, please follow the following guidelines when committing code to
 Django's Git repository:
 
-* Never change the published history of django/django branches! **Never
-  force-push your changes to django/django.** If you absolutely must (for
+* Never change the published history of `django/django` branches! **Never
+  force-push your changes to `django/django`.** If you absolutely must (for
```
Comment (akki) [inline docs/internals/contributing/committing-code.txt]: Just noticed that the `**` on this line are not making any difference. Is there any reason for having them here that I missed?

File: docs/releases/1.2.4.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ Backwards incompatible changes
 Restricted filters in admin interface
 -------------------------------------
 
-The Django administrative interface, django.contrib.admin, supports
+The Django administrative interface, `django.contrib.admin`, supports
```
Comment (timgraham) [inline docs/releases/1.2.4.txt]: module names typically go in double backticks

File: docs/releases/1.1.txt [hunk 3]
```diff
@@ -440,7 +440,7 @@ What's next?
 
 We'll take a short break, and then work on Django 1.2 will begin -- no rest for
 the weary! If you'd like to help, discussion of Django development, including
-progress toward the 1.2 release, takes place daily on the django-developers
+progress toward the 1.2 release, takes place daily on the `django-developers`
```
Comment (timgraham) [inline docs/releases/1.1.txt]: Use |django-developers| and delete the URL

File: docs/ref/contrib/gis/install/index.txt [hunk 4]
```diff
@@ -325,7 +325,7 @@ Fink
 
 `Kurt Schwehr`__ has been gracious enough to create GeoDjango packages for users
 of the `Fink`__ package system. `Different packages are available`__ (starting
-with "django-gis"), depending on which version of Python you want to use.
+with "`django-gis`"), depending on which version of Python you want to use.
```
Comment (timgraham) [inline docs/ref/contrib/gis/install/index.txt]: For packages, I usually use double backticks (and remove the quotes).

**Pull request merged**